test/companies/jobs_test.exs

```diff
@@ -14,6 +14,33 @@ defmodule Companies.JobsTest do
       insert_list(2, :job)
       assert 2 == length(Jobs.all())
     end
+
+    test "does not retrieve deleted records" do
+      insert(:job, is_deleted: true)
+      present_job = insert(:job, is_deleted: false)
+      another_present_job = insert(:job)
+
+      # TODO: rewrite me, but how?
+      job1_id = present_job.id
+      job2_id = another_present_job.id
+      assert [%Job{id: ^job1_id}, %Job{id: ^job2_id}] = Jobs.all()
+    end
+  end
+
+  describe "get!/1" do
+    test "retrieves by id" do
```

Comment:
What do you think of this?

```elixir
test "retrieves by id" do
  %{id: job_id} = insert(:job)

  assert %{id: ^job_id} = Jobs.get!(job_id)
end
```

1. We can, and I personally prefer to, use pattern matching to obtain the job's `id`
1. We can match on plain map. I don't think the test gains or loses anything using `%Job{}`, save keystrokes?